charts/rancher-istio/0.1.2/Chart.yaml

```diff
@@ -1,20 +1,17 @@
 apiVersion: v1
 name: rancher-istio
-version: 0.1.1
```

Comment:
We should match the helm chart version with the Istio version. It'll make it easier for users to understand.

Reply:
I agree. Previous istio version are used as 0.1.1 so I just make it more compatible.

Reply:
@StrongMonkey Maybe we shouldn't change it as the UI already displays it with the app version and the Istio version. 
<img width="606" alt="Screen Shot 2019-12-20 at 9 46 42 AM" src="https://user-images.githubusercontent.com/11410997/71279497-c8491080-230d-11ea-874b-bbc09512e8e6.png">